.buildkite/release-automation/pre_release.rayci.yml

```diff
@@ -8,14 +8,12 @@ steps:
     commands:
       - bazel run //ci/ray_ci/automation:weekly_green_metric -- --check
 
-  - block: "Trigger tests & image builds"
-    depends_on:
-      - step: check-release-blockers
-        allow_failure: true
-
   - label: "Check commit hash"
     key: check-commit-hash
     job_env: forge
+    depends_on:
+      - step: check-release-blockers
+        allow_failure: true
```

Comment:
we do not really allow this fancier version of `depends_on`

could you:

- just write `depends_on: check-release-blockers`
- add `allow_dependency_failure: true` (if this is not allowed in `rayci`, then add it in)

Reply:
done!
Triggered a build: https://buildkite.com/ray-project/release-automation/builds/193
Even though checking release blockers failed, it was able to proceed and trigger tests